controllers/users.js

```diff
@@ -64,7 +65,71 @@ const getUserById = async (req, res) => {
 
 const getUsers = async (req, res) => {
   try {
-    const { allUsers, nextId, prevId } = await userQuery.fetchUsers(req.query);
+    const combinations = req.query.q.split(" "); // split the query string by whites-pace
+    const qualifiers = {};
+    let allPRs;
+
+    // iterate through the combinations and parse the qualifiers
+    combinations.forEach((combination) => {
+      const [qualifier, value] = combination.split(":");
+      qualifiers[qualifier] = value;
+    });
+
```

Comment:
Should we create a utils function for this? So we can reuse this pattern across other modules too?

Reply:
sure, will shift it to utils